app/styles/style.css

```diff
@@ -1,4 +1,4 @@
-size:10px;line-height:14px}.email-item .subline{display:block}.email-item .subline-from{float:left}.email-item .subline-time{float:right}@media (max-width: 800px){.email-item .subline-time{display:none}}.email-container{padding-top:2.8em}.email-toolbar{position:fixed;z-index:10;width:100%;background-color:#fff}.email-content{position:inherit;z-index:1;margin:1em;padding-top:2.8em}.main-content{margin:2.8em 2em 0}.main-content a,.main-content a:visited{color:#4994ce}.main-content a:hover{color:#265f8b}.toolbar{margin:0;padding:0.2em 0;height:2.4em;display:-webkit-box;display:-moz-box;display:-ms-flexbox;display:-webkit-flex;display:flex;flex-flow:row nowrap;-webkit-align-items:center;align-items:center;list-style:none;font-size:14px}.toolbar li{padding:0 0.5em}.toolbar .fa{vertical-align:top;margin-right:4px;font-size:20px}.toolbar .fa.fa-mobile-smaller{font-size:16px;line-height:20px}.toolbar a{padding:0.3em 0.2em;text-decoration:none}.toolbar a,.toolbar a:visited{color:rgba(255,255,255,0.8)}.toolbar a:hover{color:#fff}.toolbar a.disabled{color:#8c8c8c !important;cursor:default}.toolbar.on-light a{color:#505050}.toolbar.on-light a:hover{color:#4994ce}.toolbar.on-light a.active{color:#265f8b}.toolbar-right{position:fixed;top:0;right:0}.dropdown-container{position:relative}.dropdown-trigger:focus{outline:none}.dropdown-trigger.open{color:#4994ce !important}.dropdown-trigger.open+.dropdown-menu{display:block}.dropdown-menu{display:none;position:absolute;top:2em;left:0;z-index:10;padding:0 0.5em;min-width:10em;list-style:none;background-color:#fff;box-shadow:0 2px 2px 0px rgba(50,50,50,0.1)}.dropdown-menu li{padding:0.4em 0}.slide-container{overflow:hidden;max-height:0px;-webkit-transition:max-height 500ms ease-in-out;-moz-transition:max-height 500ms ease-in-out;transition:max-height 500ms ease-in-out}.slide-container.open{max-height:1000px}.code{background-color:#fafafa;border:1px solid #dcdcdc;padding:1em 1em}code{background-color:#fafafa;border:1px solid #dcdcdc;padding:0.1em 0.3em}.search-container{position:relative;padding:0.3em 0.8em 0}.search-icon{position:absolute;top:0.7em;left:1.3em;color:rgba(255,255,255,0.8)}.search-input{display:block;margin:0;padding:0.3em 0.2em;width:98%;text-indent:1.9em;background-color:#4994ce;border:0;color:#fff;-webkit-font-smoothing:antialiased}.search-input:focus{outline:none;background-color:#307ab4}.search-input:focus+.search-icon{color:#fff}::-webkit-input-placeholder{-webkit-font-smoothing:antialiased;color:rgba(255,255,255,0.8);-webkit-transition:color 250ms linear;transition:color 250ms linear}::-moz-placeholder{color:rgba(255,255,255,0.8);-moz-transition:color 250ms linear;transition:color 250ms linear}::-ms-placeholder,::placeholder{-webkit-font-smoothing:antialiased;color:rgba(255,255,255,0.8);transition:color 250ms linear}.preview-iframe{border:0;width:100%;min-height:600px;position:relative}.email-item.current{background:#eaf2f9}.autoshow-toggle{float:right;cursor:pointer;color:white;padding-top:3px}
```

Comment:
We shouldn't need so many icons

Reply:
I just used `yarn css` to generate css. those icons were included by that command. Is there is any other way to generate the css file ?